redbot/cogs/trivia/data/lists/worldcup.yaml

```diff
@@ -432,3 +456,249 @@ In 2018, Italy failed to qualify after losing to *who* 1-0?:
 - Sweden
 In 2002, Netherlands failed to qualify after losing to Republic of Ireland. What was the goal difference in this qualification decider?:
 - 1
+Which is the only African nation to reach the semi-finals of the World Cup?:
+- Morocco
+In 2022, Morocco beat Spain and *who* in the knockout stages, to reach the semi-final?:
+- Portugal
+Who has scored the most goals at a single World Cup? (13):
+- Just Fontaine
+- Fontaine
+During the penalty shootout of the 2022 final, who scored the final penalty that ensured victory over the French?:
+- Gonzalo Montiel
+- Montiel
+In 2022, Netherlands played Argentina in the quarter-finals. The game broke a record for the most yellow cards shown in a FIFA World Cup match. How many were shown?:
+- 18
+- Eighteen
+Who was Costa Rica's keeper when they lost 7-0 against Spain in 2022?:
+- Keylor Navas
+- Navas
+How many red cards have ever been given out in FIFA World Cup **finals**?:
+- 5
+- Five
+Prior to Kylian Mbappé's hatrick in the final of 2022, who was the only player to have scored a hattrick in a FIFA World Cup final?:
+- Geoff Hurst
+- Hurst
+
+#### APPEARANCES
+
+Which is the most northerly European nation who has never qualified for a FIFA World Cup?:
+- Finland
+Iceland have 1 FIFA World Cup appearance, but in which year?:
```

Comment:
This structure is a little wordy, maybe
`<Country> has had only 1 FIFA World Cup appearance. In what year did they appear?`